package-lock.json

```diff
@@ -1,2268 +1,8 @@
 {
```

Comment:
Don't include package-lock to commit and don't even change package-lock using `npm ci`